src/locales/en.json

```
@@ -297,7 +297,10 @@
       "nominal_rpc_description": "The nominal compensation rate provided by DAO when there is no burnt portion in the secondary issuance. The real compensation rate is always higher than the nominal compensation rate.",
       "secondary_issuance_description": "The secondary issuance is automatically divided into three parts, the compensation, the mining reward and the treasury (burnt by now). They are proportional to the DAO deposit, the storage occupation, and the rest of CKBytes.",
       "inflation_rate_description": "Nominal inflation rate: the inflation introduced by the the primary issuance and the secondary issuance. \nNominal APC: the anti-dilution compensation rate of Nervos DAO.\nReal inflation rate: the compound inflation rate that nominal inflation rate minus the nominal APC, which is gradually to zero.",
-      "node_count_in_total": "in total"
+      "node_count_in_total": "in total",
+      "ckb_amount": "CKB Amount",
+      "contract_resource_distributed": "Contract Resource Distributed",
+      "contract_resource_distributed_description": "The x axis is contract's unique address count, the y axis is the contract's ckb amount, the symbol size is the contract's transaction count."
```

```suggestion
      "contract_resource_distributed": "Contract Resource Distribution",
      "contract_resource_distributed_description": "The x axis represets contract's unique address count, the y axis represets the contract's CKB amount, the symbol size represets the contract's transaction count."
```

I will suggest using `distribution` because there's a chart named `Epoch time distribution` at https://explorer.nervos.org/charts/epoch-time-distribution